plugins/catalog-backend-module-gitlab/src/providers/GitlabOrgDiscoveryEntityProvider.test.ts

```diff
@@ -543,6 +543,42 @@ describe('GitlabOrgDiscoveryEntityProvider', () => {
     );
   });
 
+  it('fail with scheduler but no group config when host is gitlab.com', () => {
+    const scheduler = {
+      createScheduledTaskRunner: (_: any) => jest.fn(),
+    } as unknown as PluginTaskScheduler;
+    const config = new ConfigReader({
+      integrations: {
+        gitlab: [
+          {
+            host: 'test-gitlab',
+            apiBaseUrl: 'https://api.gitlab.example/api/v4',
```

Comment:
Perhaps this should contain `gilab.com` too?

Reply:
Yes ! Nice catch thanks

Reply:
Should I also update the error condition in the plugin then ? So that it checks for the host in the config too and not just the host in the provider

Reply:
No, I don't think so. This is jus so the test reflects the functionality that the unit test is testing

Reply:
This should part of the statement 
![image](https://github.com/angom1/backstage/assets/102025968/f7e52281-18cd-4b5b-a9a3-7825c886151b)

Reply:
Yeah, that looks fine as is